frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx

```diff
@@ -18,12 +18,11 @@ import './breakdown-card.scss';
 
 const LinkableLegend: React.FC<LinkableLegendProps> = (props: LinkableLegendProps) => {
   const { metricModel, datum } = props;
+  const href = metricModel
+    ? resourcePathFromModel(metricModel, datum.link)
+    : '/dashboards/object-service';
```

Comment:
This should be not like this, we are setting the link already at one place.
This will override